### PR TITLE
com_039_rw0 upload table to Carto & S3. Add myself to readme.

### PR DESCRIPTION
**Pull request by chrowe**
## Checklist for Reviewing a Pre-Processing Script

- [ ] Does the python script contain the following 4 sections: Download data and save to your data directory, Process data, Upload processed data to Carto/Upload processed data to Google Earth Engine, Upload original data and processed data to Amazon S3 storage?
- [ ] Does the script have the standardized variable names for: dataset_name, raw_data_file, processed_data_file?
- [ ] Does the script create and use a 'data' directory?
- [ ] Does the script use a python module to automatically download the data? If this is not possible, are there explicit instructions for how to download the data (step by step with instructions about every input parameter and button to click to find the exact data) and does it use shutil to move the data from 'Downloads' into the data directory?
- [ ] Is the script automated as much as possible to minimize rewriting code the next time the dataset updates (ex 1: can you automatically pull out column names instead of typing them yourself? ex 2: if you are dropping columns with no data, did you use pandas to find the nodata columns instead of dropping the column by name?)
- [ ] Are there comments on almost every line of code to explicitly state what is being done and why?
- [ ] Are you uploading to the correct AWS location?
- [ ] For GEE scripts, did you explicitly define the band manifest with a pyramiding policy so that we can easily change it later if we need to?
- [ ] Does the README contain all relevant links?
- [ ] Does the README state the original file type?
- [ ] Does the README list all of the processing steps that were taken in the script?
- [ ] For netcdfs, does the README state which variables were pulled from the netcdf?
- [ ] Did you use the util functions whenever possible?
- [ ] Have you checked the processed data file on your computer to make sure it matches the data you uploaded to Carto? (spaces or symbol names in column titles often get changed in the upload process-please change these before uploading so that the backed up processed data matches the data on Carto)
- [ ] Does the folder name and python script file name match the dataset_name variable in the script? Are they all lowercase? Does the processing script end in '_processing.py'? (this part is often forgotten)

### CODE REVIEW

File: com_039_rw0_agricultural_trade_statistics/README.md [hunk 1]
```diff
@@ -18,4 +18,4 @@ You can view the processed {Resource Watch public title} dataset [on Resource Wa
 
 You can also download the original dataset [directly through Resource Watch]({s3 link if available}), or [from the source website](https://www.fao.org/faostat/en/#data/TCL).
```
Comment (weiqi-tori) [inline com_039_rw0_agricultural_trade_statistics/README.md:19]: Update the Python script link and s3 link. The s3 link has to be with 'https://', 'http://' won't work.

File: com_039_rw0_agricultural_trade_statistics/com_039_rw0_agricultural_trade_statistics_processing.py [hunk 2]
```diff
@@ -78,3 +83,44 @@
...
+# authenticate carto account
+CARTO_USER = os.getenv('CARTO_WRI_RW_USER')
+CARTO_KEY = os.getenv('CARTO_WRI_RW_KEY')
+set_default_credentials(username=CARTO_USER, base_url="https://{user}.carto.com/".format(user=CARTO_USER),api_key=CARTO_KEY)
+
+# upload data frame to Carto
+to_carto(df, dataset_name + '_edit', if_exists='replace', privacy="link")
+
+# set privacy to 'link' so table is accessible but not published
+update_privacy_table(dataset_name + '_edit', 'link')
```
Comment (weiqi-tori) [inline com_039_rw0_agricultural_trade_statistics/com_039_rw0_agricultural_trade_statistics_processing.py:101]: Not sure if `to_carto` has the parameter `privacy`, I didn't see it on their [reference](https://carto.com/developers/cartoframes/reference/). If so, we don't need to run `update_privacy_table`. If not, we should remove that from the line above.

File: com_039_rw0_agricultural_trade_statistics/README.md (hunk 1 above)
Comment (chrowe) [reply com_039_rw0_agricultural_trade_statistics/README.md]: resolved in 91dc2c295629e2559ca7593ac6e7df8aa35b58d1

Comment (chrowe) [reply com_039_rw0_agricultural_trade_statistics/README.md]: The spreadsheet note says "Please link to the unprocessed data file." Is that not the case any more?

File: com_039_rw0_agricultural_trade_statistics/com_039_rw0_agricultural_trade_statistics_processing.py (hunk 2 above)
Comment (chrowe) [reply com_039_rw0_agricultural_trade_statistics/com_039_rw0_agricultural_trade_statistics_processing.py:101]: Right, I was experimenting with that option but it didn't work. Removed in 91dc2c295629e2559ca7593ac6e7df8aa35b58d1

File: com_039_rw0_agricultural_trade_statistics/README.md (hunk 1 above)
Comment (weiqi-tori) [reply com_039_rw0_agricultural_trade_statistics/README.md]: It is the case. We should link to the raw data zip file on s3.

**Pull request merged**